be/api/po-app/src/main/java/cz/covid/po/api/helper/PhoneNormalizerHelper.java

```diff
@@ -0,0 +1,18 @@
+package cz.covid.po.api.helper;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class PhoneNormalizerHelper {
+    private static final String DOUBLE_ZERO = "00";
+
+    public String normalizePhone(final String originalPhone) {
+        String normalizedPhone = originalPhone.trim();
+
+        if (DOUBLE_ZERO.equals(normalizedPhone.substring(0, 2))) {
```

Comment:
```suggestion
        if (normalizedPhone.length() > 1 && DOUBLE_ZERO.equals(normalizedPhone.substring(0, 2))) {
```
Jinak upadne na StringIndexOutOfBoundsException pro "0" a pod.

Reply:
Dík, pupdatnuto, přidal jsem tam ještě null check